backend/lcfs/services/redis/lifetime.py

```diff
@@ -10,22 +10,25 @@
 
 async def init_redis(app: FastAPI) -> None:
     """
-    Creates connection pool for redis.
+    Initializes the Redis client and tests the connection.
 
-    :param app: current fastapi application.
+    :param app: current FastAPI application.
     """
     try:
-        app.state.redis_pool = ConnectionPool.from_url(
-            str(settings.redis_url),
-            encoding="utf8",
+        # Initialize Redis client directly
```

Comment:
Need to add the values for redis here because the pool configuration was removed.

Reply:
Not sure what you mean, the Redis instance handles its own connections under the hood.